app/models/user.rb

```diff
@@ -51,6 +51,15 @@ def name_or_email
     email
   end
 
+  def service_user_supervisor?
+    FeatureFlags.basic_user_roles.enabled? && supervisor? && !can_manage_others?
+  end
+
+  # A dev only user state for debugging in staging, not found in production environments
+  def admin_user_supervisor?
+    FeatureFlags.basic_user_roles.enabled? && supervisor? && can_manage_others?
```

Comment:
If a supervisor can already see the performance tracking tab on all environments then I don't see the point of the  `can_manage_others` check. By enabling it for user managers on staging we have to be referring to user managers with the caseworker role

Reply:
It pairs with the `service_user_manager?` method that checks that the user is not a user manager.

Those who can see the tab are:
Service supervisors
Admin supervisors AND service access turned on

Everyone else cannot see the nav link or tab content

Reply:
But on staging what would be the point of setting yourself to be a user manager if you're already a supervisor and therefore should be able to see the page?